contrib/scripts/createmodule.py

```diff
@@ -36,10 +36,21 @@
     parser.print_usage()
     exit(1)
 
+# Determine if running environment is based on cmd.exe or not
+def iscmdshell():
+    import platform
```

Comment:
This `import` statement should be placed with the other `import` calls at top of script, as this function is likely to be called each time the script is run